dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -20,20 +20,21 @@ jwtSso.desc=Information about configuring JWT Single Sign On.
 cookieName=Name of cookie
 cookieName.desc=Name of the cookie that is used to store the JWT token.
 
-setCookiePathToWebAppContextPath=Set cookie path to context path
-setCookiePathToWebAppContextPath.desc=Set the cookie path of the SSO cookie to the context path of the application that is being authenticated, which limits the use of this cookie to only this app.
+setCookiePathToWebAppContextRoot=Set cookie path to context path
+setCookiePathToWebAppContextRoot.desc=Set the cookie path of the JWT cookie to the context path of the application that is being authenticated, which limits the use of this cookie to only this app.
 
 includeLtpaCookie=Include LTPA cookie
-includeLtpaCookie.desc=After successful authentication with a JWT token, include an LTPA cookie in addition to the SSO cookie.
+includeLtpaCookie.desc=After successful authentication with a JWT token, include an LTPA cookie in addition to the JWT cookie.
 
-fallbackToLtpa=Fall back to LTPA cookie
-fallbackToLtpa.desc=If the SSO cookie is missing or invalid, attempt to process an LTPA cookie.
+#initially this property was exposed as fallbackToLtpa but now is useLtpaIfJwtAbsent
+fallbackToLtpa=Use LTPA cookie if JWT cookie is not present
+fallbackToLtpa.desc=If the JWT cookie is missing or invalid, attempt to process an LTPA cookie if present. 
```

Comment:
I thought we'll only try to process the LTPA cookie if the JWT cookie is missing; if the JWT cookie is invalid, it was my understanding that we would immediately fail and return a 401 without falling back to LTPA.

Reply:
yep, you're right, Ut has updated SsoAuthenticator.  I'll update the props file.